fhir-mapping/src/main/ml-modules/root/data-services/utils/opticQueryUtils.sjs

```diff
@@ -0,0 +1,75 @@
+
+
+const beginningOfDay = xs.time('00:00:00')
+
+function applyStringParamModifier(modifier, value) {
+  if("exact" === criteria.modifier) {
```

Comment:
I don't see a parameter for this function or a module-level variable named `criteria`, should this (and the other conditions) be using `modifier` instead?

Reply:
You are correct, this should be resolved.